extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -685,3 +687,20 @@ def validate(self, classifier_data):
                     raise utils.ValidationError(
                         'Expected values of %s to be a int, received %s' % (
                             list_property, value))
+
+        for list_of_list_property in list_of_list_properties:
+            if list_of_list_property not in classifier_data:
+                raise utils.ValidationError(
+                    'Expected %s to be a key in classifier_data' %
+                    list_of_list_property)
+            if not isinstance(classifier_data[list_of_list_property], list):
+                raise utils.ValidationError(
+                    'Expected %s to be a list, received %s' % (
+                        list_of_list_property,
+                        classifier_data[list_of_list_property]))
+            for inner_list in classifier_data[list_of_list_property]:
+                for value in inner_list:
+                    if not isinstance(value, int):
```

Comment:
Not all the lists have elements of type int. [_b_dl_](https://github.com/pranavsid98/oppia/blob/e088b9c0c1a0f471b34ffe4ef7143ba032ad0d19/extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py#L139) is a list of booleans for example.